force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +353,15 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+        String word = '';
         // Loop through the list of words
+        for (Integer i = 0; i < words.size(); i++) {
```

Comment:
Can you find the List method that will do the same as this question in one line?